pkg/templates/k8sresource_funcs.go

```diff
@@ -38,7 +38,7 @@ func (t *TemplateResolver) fromSecret(namespace string, secretname string, key s
 	sEnc := base64.StdEncoding.EncodeToString(keyVal)
 
 	// add this Secret to list of  Objs referenced by the template
-	t.addToReferencedObs(secret.APIVersion, secret.Kind, secret.Namespace, secret.Name)
+	t.addToReferencedObs("/v1", "Secret", secret.Namespace, secret.Name)
```

Comment:
Could you please rename this method to `addToReferencedObjs` so it matches the struct field name?